perception/map_based_prediction/src/path_generator.cpp

```diff
@@ -400,10 +401,90 @@ FrenetPoint PathGenerator::getFrenetPoint(const TrackedObject & object, const Po
     static_cast<float>(tf2::getYaw(ref_path.at(nearest_segment_idx).orientation));
   const float delta_yaw = obj_yaw - lane_yaw;
 
+  const float ax =
+    (use_vehicle_acceleration_)
+      ? static_cast<float>(object.kinematics.acceleration_with_covariance.accel.linear.x)
+      : 0.0;
+  const float ay =
+    (use_vehicle_acceleration_)
+      ? static_cast<float>(object.kinematics.acceleration_with_covariance.accel.linear.y)
+      : 0.0;
+
+  // The decay constant λ = ln(2) / exponential_half_life
+  // a(t) = acc - acc(1-e^(-λt)) = acc(e^(-λt))
+  // V(t) = Vo + acc(1/λ)(1-e^(-λt))
+  // x(t) = Xo + Vo * t + t * acc(1/λ) + acc(1/λ^2)(e^(-λt)-1)
+  // x(t) = Xo + (Vo + acc(1/λ)) * t  + acc(1/λ^2)(e^(-λt)-1)
+  // acceleration_distance = acc(1/λ) * t  + acc(1/λ^2)(e^(-λt)-1)
```

Comment:
Can these comments be removed ? Each function is recalled when used in the code. You may also want to add a citation to where these equations come from in the docstring of the function.